packages/devtools_app/lib/src/screens/profiler/cpu_profile_transformer.dart

```diff
@@ -90,6 +99,8 @@ class CpuProfileTransformer {
       '(${cpuProfileData.cpuProfileRoot.inclusiveSampleCount})',
     );
 
+    await cpuProfileData.computeBottomUpRoots();
```

Comment:
this means that we will always compute the bottom up roots even when we don't need them. Before, it looked like we were lazily computing the bottom up roots only when the user opened the bottom up table, right? We may still want to do compute this lazily to save on initial load time.

Reply:
Yes, it is eager now, but I think the initial screen is always the bottom up screen after loading a profile, so we will always compute this?

Reply:
Another option could be making the `bottomUpRoots` getter asynchronous, not sure how that might affect some of the other code though, but I could look into it.

Reply:
Ah yes good point. Fine to leave as is. Maybe add a comment that says this is fine because Bottom Up is the initial view for the profiler, otherwise this should be computed lazily?